src/components/Table/TableHead/TableHead.test.jsx

```diff
@@ -285,6 +285,43 @@ describe('TableHead', () => {
       expect(myActions.onChangeOrdering).toHaveBeenCalledWith(orderingAfterTogleShow);
     });
 
+    it('toggle show column without initial width correctly updates the column widths of visible columns', () => {
+      myProps.tableState = {
+        ...myProps.tableState,
+        ordering: [
+          { columnId: 'col1', isHidden: false },
+          { columnId: 'col2', isHidden: false },
+          { columnId: 'col3', isHidden: true },
+        ],
+      };
+      myProps.columns = [
+        { id: 'col1', name: 'Column 1', width: '200px' },
+        { id: 'col2', name: 'Column 2', width: '200px' },
+        { id: 'col3', name: 'Column 3' },
+      ];
+
+      mockGetBoundingClientRect.mockImplementation(() => ({ width: 200 }));
+
+      const wrapper = mount(<TableHead {...myProps} />);
```

Comment:
Are we able to use the testing library in this case?

Reply:
Yes probably, this test is almost identical to the previous test so I figured that I'd take the quick route for this fix and do a copy paste. 

Do you want me to rewrite it using testing library?

Reply:
Preferably, but I agree with taking a speedier copy/paste route for bugs where the tests are already consistently written with enzyme.